### PR TITLE
Pbi 350 stage nav options

### PR DESCRIPTION
**Pull request by edlu77**
#350

### CODE REVIEW

File: projects/ccf-rui/src/app/modules/content/content.component.ts [hunk 1]
```diff
@@ -21,19 +24,45 @@ export class ContentComponent {
...
 
+  /**
+   * Sets up ResizeSensor to listen to changes in top bar width and enables dropdown if below a certain width
+   */
+  ngAfterViewInit(): void {
+    const {
+      topBar: { nativeElement: topBar }
+    } = this;
+
+    this.sensor = new ResizeSensor(topBar, () => {
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/modules/content/content.component.ts:59]: This logic should be moved into the stage nav component

File: projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.html [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-  <mat-radio-button value="left" class="nav-input">Left</mat-radio-button>
-  <mat-radio-button value="right" class="nav-input">Right</mat-radio-button>
-  <mat-radio-button value="anterior" class="nav-input" checked>Anterior</mat-radio-button>
-  <mat-radio-button value="posterior" class="nav-input">Posterior</mat-radio-button>
-</mat-radio-group>
+<div class='dropdown-label' (click)="toggleNav()">
+  <mat-label>Options</mat-label>
+  <mat-icon>arrow_drop_down</mat-icon>
+</div>
+<div class='stage-nav' [class.hidden]='stageNavHidden'>
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.html:5]: Slightly strange class name considering this is inside the stage-nav component. A better name might be `options` or something along those lines

File: projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.scss [hunk 3]
```diff
@@ -1,20 +1,87 @@
...
+        display: inherit;
+        width: 100%;
+        margin: 0;
+        border-top: 3px solid;
+      }
+    
+      .input-group {
+        margin-left: 1.5rem;
+        margin-right: 0;
+        display: flex;
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.scss:67]: Might look better to have one option per row. I.e. `flex-direction: column`

File: projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.ts [hunk 4]
```diff
@@ -39,6 +39,18 @@ export class StageNavComponent {
...
 
+  /**
+   * Determines if stage nav settings are visible
+   */
+  stageNavHidden = true;
+
+  /**
+   * Toggles appearance of stage nav settings
+   */
+  toggleNav(): void {
```
Comment (axdanbol) [inline projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.ts:50]: Needs to close nav when clicked outside of it

**Review by axdanbol — CHANGES_REQUESTED** [review]
A few improvements needed

**Comment by axdanbol** [comment]
Investigate/consider using `mat-menu`

File: projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.scss (hunk 3 above)
Comment (edlu77) [reply projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.scss:70]: Leonard wanted two options per row, so going to leave it this way for now.

**Review by bherr2 — APPROVED** [review]

**Pull request merged**